www/js/overlay.js

```diff
@@ -1,7 +1,9 @@
+/* eslint-disable import/no-unresolved */
```

Comment:
prefer if we do `// eslint-disable-next-line import/no-unresolved` before the line